todo_app/db.py

```diff
@@ -0,0 +1,139 @@
+from pymongo import MongoClient, response
+from pymongo.errors import ConnectionFailure
+from bson.objectid import ObjectId
+import sys
+import os
+from flask import current_app as app
+
+from requests.models import Response
+
+class DBMongo:
+    def __init__(self):
+
+        try:
+            url = "mongodb+srv://" + str(os.environ['M_KEY']) + ":" + str(os.environ["M_TOKEN"]) + "@virendra-mongodb-cluste.280hx.mongodb.net/Virendra-MongoDB-Cluster0?retryWrites=true&w=majority"
```

Comment:
This currently only works for your individual cluster - instead it would be nicer to extract this entire connection string as a single environment variable so that it's easy to plug in an alternative value entirely